crates/sui-graphql-client/src/lib.rs

```diff
@@ -765,6 +965,63 @@ impl Client {
         }
     }
 
+    /// Return the contents' JSON of an object that is a Move object.
+    ///
+    /// If the object does not exist (e.g., due to prunning), this will return `Ok(None)`.
+    /// Similarly, if this is not an object but an address, it will return `Ok(None)`.
+    pub async fn object_move_contents(
```

Comment:
nit: I would probably call this `move_object_contents`, seeing as a `MoveObject` is a concept we have.